Gruntfile.js

```diff
@@ -18,14 +18,41 @@ module.exports = function( grunt ) {
   grunt.initConfig({
     pkg: grunt.file.readJSON( "package.json" ),
 
-    csslint: {
-      lax: {
+    watch: {
```

Comment:
I still wonder if this is a good idea, and based on what Jon says next week, maybe we should hold off til then on auto-building changed src files.